tests/timescaledb/copydb.sh

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+
+set -x
+set -e
+
+# This script expects the following environment variables to be set:
+#
+#  - PGCOPYDB_SOURCE_PGURI
+#  - PGCOPYDB_TARGET_PGURI
+#  - PGCOPYDB_TABLE_JOBS
+#  - PGCOPYDB_INDEX_JOBS
+
+# make sure source and target databases are ready
+pgcopydb ping --source ${PGCOPYDB_SOURCE_PGURI} --target ${PGCOPYDB_TARGET_PGURI}
+
+psql -a ${PGCOPYDB_SOURCE_PGURI} <<EOF
+create role tsdb NOSUPERUSER CREATEDB NOCREATEROLE LOGIN PASSWORD '0wn3d';
+create database tsdb owner tsdb;
+EOF
+
+# copying roles needs superuser
+# and we use the postgres database here still
+pgcopydb copy roles --source ${PGCOPYDB_SOURCE_PGURI} --target ${PGCOPYDB_TARGET_PGURI}
+
+# now create the tsdb database on the target, owned by new role tsdb
+psql ${PGCOPYDB_TARGET_PGURI} <<EOF
+create database tsdb owner tsdb;
```

Comment:
What is the purpose of this database? I don't see them being used.

Reply:
The purpose of the `tsdb` database was to simply check that the creation of the database is happening properly without affecting the data in the source and target. It had no effect on the migration, which is why I removed it.